superset-frontend/webpack.config.js

```diff
@@ -293,6 +293,7 @@ const config = {
             loader: 'less-loader',
             options: {
               sourceMap: isDevMode,
+              javascriptEnabled: true,
```

Comment:
This option is usually nested [a bit deeper](https://ant.design/docs/react/use-with-create-react-app#Customize-Theme). Are you sure it works this way?

Reply:
Yeah, I believe the docs are referring to webpack 3. It breaks when it's wrong, so you know pretty fast.